src/unix/fs.c

```diff
@@ -547,6 +547,8 @@ static int uv__fs_statfs(uv_fs_t* req) {
 
 #if defined(__sun) || defined(__MVS__)
   stat_fs->f_type = 0;  /* f_type is not supported. */
+#elif defined(__OpenBSD__)
+  stat_fs->f_type = (uint64_t)buf.f_fstypename;
```

Comment:
This seems incorrect. According to http://man.openbsd.org/4.4BSD-Lite2/statfs.2, there is a `f_type` field on OpenBSD. Furthermore, `f_fstypename` is a char array, and it's being assigned to a `uint64_t` here.

Reply:
Not sure about the man page here it points to 4.4BSD-Lite 2 system but if I point to the actual OpenBSD 6.5
http://man.openbsd.org/OpenBSD-6.5/statfs.2

There is none I did on purpose so the developer can cast it back to string afterwards but I can disable the field for OpenBSD if you disagree no worries :)

Reply:
Ah you're right about that. Sorry.

In that case, I think we should just set it to zero (meaning `__OpenBSD__` can just be part of the previous `#if`).